From 040795ecde5744661d1824f7d0571af75a5e847b Mon Sep 17 00:00:00 2001
From: Max Ihlenfeldt <max@igalia.com>
Date: Wed, 13 Dec 2023 15:15:42 +0000
Subject: [PATCH] Backport "IWYU for
 g/c/service/shared_image/ozone_image_gl_textures_holder.h"

This patch fixes a build error due to a missing include. It will be
included in upstream's 122 release.

Upstream-Status: Backport [https://crrev.com/c/5119728]
Signed-off-by: Max Ihlenfeldt <max@igalia.com>
---
 .../service/shared_image/ozone_image_gl_textures_holder.h        | 1 +
 1 file changed, 1 insertion(+)

diff --git a/gpu/command_buffer/service/shared_image/ozone_image_gl_textures_holder.h b/gpu/command_buffer/service/shared_image/ozone_image_gl_textures_holder.h
index bb82853..a49b666 100644
--- a/gpu/command_buffer/service/shared_image/ozone_image_gl_textures_holder.h
+++ b/gpu/command_buffer/service/shared_image/ozone_image_gl_textures_holder.h
@@ -6,6 +6,7 @@
 #define GPU_COMMAND_BUFFER_SERVICE_SHARED_IMAGE_OZONE_IMAGE_GL_TEXTURES_HOLDER_H_

 #include <memory>
+#include <vector>

 #include "base/memory/raw_ptr.h"
 #include "base/memory/ref_counted.h"
